src/Resources/ui/js/components/CollapsibleContent.js

```diff
@@ -19,6 +19,7 @@ class CollapsibleContent extends Component {
 
         this.content = this.vdom.querySelector(CONTENT_IDENTIFIER);
         this.title = this.vdom.querySelector(TITLE_IDENTIFIER);
+        this.element = this.vdom.querySelector(COLLAPSIBLE_CONTENT_IDENTIFIER);
```

Comment:
`this.vdom` already contains your element. There's no need for this extra selector.